test/algorithms/test_min_eigen_optimizer.py

```diff
@@ -39,7 +40,13 @@
     QuadraticProgramToQubo,
 )
 from qiskit_optimization.problems import QuadraticProgram
+from qiskit_optimization.runtime import VQEProgram, QAOAProgram
 from qiskit_optimization.translators import from_docplex_mp
+from test.optimization_test_case import (
+    QiskitOptimizationTestCase,
+    requires_extra_library,
+)
+from test.runtime.fake_vqeruntime import FakeRuntimeProvider
```

Comment:
The `test` imports must be at the top of the file (see also the test failures 🙂)

Reply:
Okay 👍  Done in e9289b58a9ad96b074fe7ab3728ef10da0f05d1c.